statistics.md

```diff
@@ -400,17 +400,31 @@ For more information on the `KILL` statement, see [`KILL`](/sql-statements/sql-s
+
 #### `tidb_build_stats_concurrency`
 
-Currently, when you run the `ANALYZE` statement, the task is divided into multiple small tasks. Each task only works on one column or index. You can use the `tidb_build_stats_concurrency` parameter to control the number of simultaneous tasks. The default value is `4`.
+When you run the `ANALYZE` statement, the task is divided into multiple small tasks. Each task only works on statistics of one column or index. You can use the `tidb_build_stats_concurrency` parameter to control the number of simultaneous small tasks. The default value is `2`. The default value is `4` for v7.4.0 and earlier versions.
+
+#### `tidb_build_sampling_stats_concurrency`
+
+When analyzing ordinary columns, you can use [`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-new-in-v750) to control the concurrency of executing sampling tasks. The default value is `2`.
+
+#### `tidb_analyze_partition_concurrency`
+
+When running the `ANALYZE` statement, you can use [`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) to control the concurrency of task writing. The default value is `2`. The default value is `1` for v7.4.0 and earlier versions.
 
 #### `tidb_distsql_scan_concurrency`
 
-When you analyze regular columns, you can use the `tidb_distsql_scan_concurrency` parameter to control the number of Region to be read at one time. The default value is `15`.
+When you analyze regular columns, you can use the [`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) parameter to control the number of Region to be read at one time. The default value is `15`. Meanwhile, it will also affect query performance, please adjust carefully.
 
 #### `tidb_index_serial_scan_concurrency`
 
-When you analyze index columns, you can use the `tidb_index_serial_scan_concurrency` parameter to control the number of Region to be read at one time. The default value is `1`.
+When you analyze index columns, you can use the [`tidb_index_serial_scan_concurrency`](/system-variables.md#tidb_index_serial_scan_concurrency) parameter to control the number of Region to be read at one time. The default value is `1`. Meanwhile, it will also affect query performance, please adjust carefully.
```

Comment:
```suggestion
When you analyze index columns, you can use the [`tidb_index_serial_scan_concurrency`](/system-variables.md#tidb_index_serial_scan_concurrency) parameter to control the number of Regions to be read at one time. The default value is `1`. Meanwhile, it will also affect query performance. Adjust the value carefully.
```